.github/workflows/go.yml

```diff
@@ -10,14 +10,14 @@ jobs:
   build:
     strategy:
       matrix:
-        go-version: [1.18.x, 1.19.x, 1.20.x]
+        go-version: [1.20.x, 1.21.x]
```

Comment:
```suggestion
        go-version: [1.19.x, 1.20.x, 1.21.x]
```

(I tend to have one more version than the official Go version)

Reply:
Sure, let me just adapt that.

Reply:
Done, thanks!